### PR TITLE
some minor improvements / cosmetics

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 
-This repository provides a tool to automatically query the website of the
-German vaccination program. With a given registration code, this tool detects
-if appointments for vaccinations become available. Different notification 
-channels (e.g. e-mail or push notification) are provided to notify the user.
+This repository provides a tool to automatically query the website of the German vaccination program.
+
+With a given registration code, this tool detects if appointments for vaccinations become available.
+
+Different notification channels (e.g. e-mail or push notification) are provided to notify the user.
```
Comment (ThomasWaldmann) [inline README.md:7]: just whitespace, easier to read.

File: README.md [hunk 2]
```diff
@@ -33,15 +34,16 @@ should be in the format
...
 
 ### Setup Browser Automation
 
 The tool is currently based on the selenium browser automation package to control
 the Chrome browser. The paths to the browser executable and the corresponding
-browser driver need to be set in the [example.py](example.py). More information on
-the installation of Selenium and the required executables can be found in
+browser driver need to be set in the [example.py](example.py).
+
+More information on the installation of Selenium and the required executables can be found in
```
Comment (ThomasWaldmann) [inline README.md:46]: more whitespace.

File: README.md [hunk 3]
```diff
@@ -54,6 +56,7 @@ The tool can take a list of notification channels. Currently this includes
  
 For each channel, specific details (e.g. target e-mail address, output-folder, etc.)
 are required to be filled into the [example.py](example.py).
+
```
Comment (ThomasWaldmann) [inline README.md:59]: same here

File: example.py [hunk 4]
```diff
@@ -14,7 +14,7 @@
 
     # List of URLs to be checked
     urls = [
-        "https://XXX-iz.impfterminservice.de/impftermine/suche/XXXX-XXXX-XXXX/YYYYY/"
+        "https://XXX-iz.impfterminservice.de/impftermine/suche/XXXX-XXXX-XXXX/YYYYY/",
```
Comment (ThomasWaldmann) [inline example.py:17]: easier to extend that way.

File: setup.py [hunk 5]
```diff
@@ -5,6 +5,19 @@
     version="0.1.0",
     author="Andreas Doerr",
     author_email="andreasdoerr@gmx.net",
+    license='MIT',
     description="Tool to automatically query impfterminservice.de",
     packages=setuptools.find_packages(),
+    python_requires='>=3.6',
```
Comment (ThomasWaldmann) [inline setup.py:11]: older python 3.x are not supported any more anyway.

Comment (ThomasWaldmann) [reply setup.py:11]: avoids that someone tries this on python2.

File: setup.py [hunk 6]
```diff
@@ -5,6 +5,19 @@
...
+    classifiers=[
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+    ],
```
Comment (ThomasWaldmann) [inline setup.py]: metadata for pypi. you can't make a package from this yet due to the way example.py needs editing, but maybe later...

File: setup.py [hunk 7]
```diff
@@ -5,6 +5,19 @@
...
     author_email="andreasdoerr@gmx.net",
+    license='MIT',
     description="Tool to automatically query impfterminservice.de",
     packages=setuptools.find_packages(),
+    python_requires='>=3.6',
+    classifiers=[
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
```
Comment (andreasdoerr) [inline setup.py]: How's this with respect to `python_requires='>=3.6'`. Not sure which min. Python version would really be necessary.

Comment (ThomasWaldmann) [reply setup.py]: oops. wait a sec...

Comment (ThomasWaldmann) [reply setup.py]: fixed.

3.6 is the minimum version supported by python development.

also, it is nice as a minimum version as it supports f-strings and some other nice features not present in older versions.

**Review by andreasdoerr — COMMENTED** [review]
Thank you!

**Pull request merged**